### PR TITLE
avoid overriding `config.js` when running tests

### PR DESCRIPTION
**Pull request by khassel**
solves #3201

### CODE REVIEW

File: tests/utils/weather_mocker.js [hunk 1]
```diff
@@ -35,9 +35,11 @@ const injectMockData = (configFileName, extendedData = {}) => {
 	} else {
 		mockWeather = readMockData("current", extendedData);
 	}
-	let content = fs.readFileSync(path.resolve(`${__dirname}../../../${configFileName}`)).toString();
+	let content = fs.readFileSync(configFileName).toString();
 	content = content.replace("#####WEATHERDATA#####", mockWeather);
-	fs.writeFileSync(path.resolve(`${__dirname}../../../config/config.js`), content);
+	const newFile = configFileName.replace(".js", "_full.js");
```
Comment (rejas) [inline tests/utils/weather_mocker.js:40]: should we name them "_temp" or "_test" maybe?

**Review by rejas — CHANGES_REQUESTED** [review]
thx. the files dont get deleted at the end. maybe delete them after the test?

File: tests/utils/weather_mocker.js (hunk 1 above)
Comment (khassel) [reply tests/utils/weather_mocker.js:40]: found no good name, so going for "_temp"

**Comment by khassel** [comment]
> maybe delete them after the test?

done, also for file `tests/configs/port_variable.js` which is created by template_spec test

**Review by rejas — APPROVED** [review]

**Pull request merged**